cockroachcloud/pci-dss.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Payment Card Industry Data Security Standard (PCI DSS) Compliance in CockroachDB Cloud Dedicated
+summary: Learn about compliance with the Payment Card Industry Data Security Standard (PCI DSS) for CockroachDB Cloud Dedicated clusters.
+toc: true
+docs_area: manage.security
+---
+
+The Payment Card Industry Data Security Standard (PCI DSS) is a series of requirements for the safe handling of sensitive data associated with credit and debit cards. In the PCI DSS standard, and in the rest of this page, this data is referred to as "cardholder data". When implemented correctly, PCI DSS helps to protect your organization and your customers from fraud. PCI DSS is mandated by credit card issuers but administered by the [Payment Card Industry Security Standards Council](https://www.pcisecuritystandards.org/).
+
+{% include v22.1/dedicated-pci-compliance.md %}
+
+Responsibility for compliance with PCI DSS is shared among multiple parties, including card issuers, banks, software-as-a-service (SaaS) providers, and retail merchants. Compliance measures are implemented as a series of business practices, security controls, and technological solutions. An organization's compliance with PCI DSS is certified by a PCI Qualified Security Assessor (QSA).
+
+This page provides information about {{ site.data.products.dedicated }}'s compliance with PCI DSS, describes some of the ways that {{ site.data.products.db }} implements and enforces compliance, and illustrates some of the types of changes you may need to implement outside of your {{ site.data.products.dedicated }} clusters. For detailed instructions and working examples, refer to [Integrate {{ site.data.products.dedicated }} with Satori](/docs/{{site.versions["stable"]}}/satori-integration.html).
+
+## Overview of PCI DSS
+
+When a system complies with PCI DSS, the system meets the goals of the standard by implementing a series of twelve requirements, according to a PCI Qualified Security Assessor (QSA). The following table, which is published in Payment Card Industry Security Standards Council's [PCI DSS Quick Reference Guide, version 3.2.1](https://listings.pcisecuritystandards.org/documents/PCI_DSS-QRG-v3_2_1.pdf), summarizes the goals and requirements of PCI DSS.
+
+<table>
+<tgroup cols="2">
+<thead>
+<tr>
+  <th>Goal</th>
+  <th>PCI DSS Requirement</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+  <td>Build and Maintain a Secure Network and Systems</td>
```

Comment:
These are well and good for a first pass, but it would be great to expand these and relate them to CRDB and CC features that help to address the requirements. I could see either you or myself taking a pass at that a bit down the road when have a clearer scoping of what the complete set of features will be when we roll stuff out ~January

Reply:
I created a ticket and assigned it to myself, although it would be great if you could take it on @mdlinville as you have already acquired some context on it.